client/src/pages/Dashboard/MentorProfile.jsx

```diff
@@ -123,7 +123,7 @@ const Student = () => {
             <p>No Mentor data available.</p>
           )}
           <div className=" justify-center items-center">
-            {user?.email === "kumarvivek@gmail.com" && (
```

Comment:
don't remove "kumarvivek@gmail.com" 

use the OR operator and add user?.role === "Admin"